src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -22,10 +23,17 @@
 def get_file_paths(directory):
     logger.debug(locals())
     file_paths = []
-    for root, _, files in os.walk(directory):
-        for file in files:
-            if file.split("/")[-1] == "qna.yaml":
-                file_paths.append(os.path.join(root, file))
+    root_path = Path(directory)
+    root_paths = [
+        entry
+        for entry in root_path.iterdir()
+        if entry.is_dir() and entry.name != "knowledge"
```

Comment:
We still need feedback on whether the skill dirs are written in stone, but can we go ahead and filter out . directories (so we don't include .git and .github), scripts, and docs.  I don't think we will find a qna.yaml in there, but just in case

Reply:
```if entry.is_dir() and not entry.name.startswith(".") and entry.name not in ["knowledge", "scripts", "docs"]```